build.sbt

```diff
@@ -145,8 +145,8 @@ lazy val V = new {
   val bloop = "1.3.2"
   val sbtBloop = bloop
   val gradleBloop = bloop
-  val scalafmt = "2.0.0"
-  val scalafmtDynamic = "2.0.0-RC4" // Not v2.0.0 because it has a regression https://github.com/scalameta/scalafmt/issues/1442
+  val scalafmt = "2.0.1"
+  val scalafmtDynamic = "2.0.1"
```

Comment:
let's remove this variable and only use scalafmt (it was like this before the regression)

Reply:
fixed https://github.com/scalameta/metals/pull/863/commits/1f7379a8db70cf8568303bf110b4a899d587a55b 👍